### PR TITLE
angular 5 upgrade

### PR DESCRIPTION
**Pull request by BenevidesLecontes**

### CODE REVIEW

**Comment by kirakishin** [comment]
hi!
did you test with angular 5.0.0 ? i'am facing to an issue on runtime : 
`No NgModule metadata found for 'function (){}'`

but may i don't know yet if its from uirouter or not.
Now, iam waiting for your angular 5 migration :)

**Comment by christopherthielen** [comment]
@kirakishin latest ui-router (1.0.0-beta.9) works with angular 5.  You can see the [sample app](https://github.com/ui-router/sample-app-angular/) for an example

File: yarn.lock [hunk 1]
```diff
@@ -2499,9 +2532,9 @@ karma-webpack@^2.0.2:
     source-map "^0.1.41"
     webpack-dev-middleware "^1.0.11"
 
-karma@^1.5.0:
-  version "1.6.0"
-  resolved "https://registry.yarnpkg.com/karma/-/karma-1.6.0.tgz#0e871d4527d5eac56c41d181f03c5c0a7e6dbf3e"
+karma@^1.7.1:
+  version "1.7.1"
+  resolved "http://ci.dmrj.com.br:7001/karma/download/karma-1.7.1.tgz#85cc08e9e0a22d7ce9cca37c4a1be824f6a2b1ae"
```
Comment (christopherthielen) [inline yarn.lock]: what is `ci.dmrj.com.br`?

**Comment by mkoczorowski** [comment]
it does work with 5. I can confirm this. I have upgraded on the release date. Well done!

**Comment by kirakishin** [comment]
Hello, 
my issue was apparently related to the angular compiler 1.5, it doesn't like manual bootstrap separated into 2 statements : 
```typescript
const platform = platformBrowserDynamic();
platform.bootstrapModule(AppModule);
```

i must use : 
```typescript
platformBrowserDynamic().bootstrapModule(AppModule);
```

i confirm that angular 5 works with ui-router beta 9 :)

**Comment by pscanlon1** [comment]
Any idea why im getting this error?  Followed the tutorial..
 using angular 5
No provider for Router! ; Zone: <root> ; Task: Promise.then ; Value: Error: StaticInjectorError[Router]:

**Comment by christopherthielen** [comment]
`Router` is not correct when using UI-Router

**Comment by JLHwung** [comment]
I suggest this PR be closed due to lack of activity. It is better to move forward and focus on Angular 6 now.

**Pull request closed without merging**